shielddash/core/static/js/app/components/containers/percent-toggle-container.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PercentToggle from '../views/percent-toggle';
+import store from '../../store';
+import {showPercentValues, hidePercentValues} from '../../actions/study-actions';
+
+
+const PercentToggleContainer = React.createClass({
+  _togglePercentage: function(evt) {
+    let study = document.querySelector('.study-wrapper');
+    if (study) {
+      study.querySelector('.percent-toggle .switch').classList.toggle('active');
+      study.classList.toggle('show-percentage');
+    }
+    if (study.classList.contains('show-percentage')) {
```

Comment:
Should this be inside the `if (study)` block on line 11? If that block evaluates to false, wouldn't this throw an exception?